lib/cache/unprocessedSubmissionsCache.ts

```diff
@@ -0,0 +1,42 @@
+import { logMessage } from "@lib/logger";
+import { getRedisInstance } from "../integration/redisConnector";
+
+// Return a random number between 180 and 300  (3 minutes <-> 5 minutes)
+const randomCacheExpiry = () => Math.floor(Math.random() * 180 + 120);
+
+export const numberOfUnprocessedSubmissionsCacheCheck = async (
+  formID: string
+): Promise<number | null> => {
+  const checkParameter = `unprocessed-submissions:${formID}`;
```

Comment:
Most of our cache keys start with the object name; form, auth, setting.  What do you think about renaming the key to `form:submissions:unprocessed:${formID}`?  This leaves room in case in the future we want to continue to add values to the cache like `form:submissions:all:${formID}` or `form:submissions:processed:${formID}`, etc.

Reply:
Yep that works!